build/Dockerfile.functest

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.15.2 AS builder
+
+WORKDIR /go/src/github.com/kubevirt/hyperconverged-cluster-operator/
+COPY . .
+RUN make build-functest
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal
+
+ENV USER_UID=1001 \
+    TEST_OUT_PATH=/test
+
+COPY --from=builder --chown=${USER_UID}:0 /go/src/github.com/kubevirt/hyperconverged-cluster-operator/tests/func-tests/_out/func-tests.test  ${TEST_OUT_PATH}/
+COPY --from=builder --chown=${USER_UID}:0 /go/src/github.com/kubevirt/hyperconverged-cluster-operator/hack  ${TEST_OUT_PATH}/hack
+COPY --from=builder --chown=${USER_UID}:0 /go/src/github.com/kubevirt/hyperconverged-cluster-operator/deploy  ${TEST_OUT_PATH}/deploy
+COPY --from=builder --chown=${USER_UID}:0 /go/src/github.com/kubevirt/hyperconverged-cluster-operator/cluster ${TEST_OUT_PATH}/cluster
+
+WORKDIR ${TEST_OUT_PATH}
+ENTRYPOINT  ./hack/run-tests.sh
+USER ${USER_UID}
```

Comment:
Please move these above the copy commands, because these lines are not going to be changed so frequentlly, while the `COPY` line will never be cached and will always crate new layer.

Reply:
DONE!!!